src/core/Akka/Routing/RoutedActorCell.cs

```diff
@@ -167,7 +167,7 @@ public override void Start()
                 var deprecatedPaths = group.Paths;
 
                 var paths = deprecatedPaths == null
-                        ? group.GetPaths(System).ToArray()
+                        ? group.GetPaths(System)?.ToArray()
```

Comment:
The next line is call to `paths.NonEmpty()` - if we're accepting to have `paths` to be potentially null, this could end with NRE.